src/main/include/RobotContainer.h

```diff
@@ -312,90 +367,164 @@ class RobotContainer
     std::vector<frc::Pose2d> close4Waypoint2
     {
       frc::Pose2d(2.65_m, 5.55_m, 180_deg),
-      frc::Pose2d(2_m, 6.8_m, 180_deg),
-      frc::Pose2d(2.65_m, 7_m, 180_deg)
+      frc::Pose2d(2.65_m, 7_m, 230_deg)
     };
 
     std::vector<frc::Pose2d> close4Waypoint3
     {
-      frc::Pose2d(2.65_m, 7_m, 180_deg),
-      frc::Pose2d(2_m, 4.3_m, 180_deg),
+      frc::Pose2d(2.65_m, 7_m, 230_deg),
+      frc::Pose2d(2.2_m, 4.3_m, 180_deg),
       frc::Pose2d(2.6_m, 4.1_m, 180_deg),
-      frc::Pose2d(2.35_m, 4.1_m, 180_deg)
+      frc::Pose2d(2.35_m, 4.1_m, 130_deg)
     };
 
     std::vector<units::meters_per_second_t> close4PointSpeed1
     {
-      0_mps,
+      0.5_mps,
       2_mps
     };
 
     std::vector<units::meters_per_second_t> close4PointSpeed2
     {
-      1_mps,
-      2_mps,
+      1.5_mps,
       1.5_mps
     };
 
     std::vector<units::meters_per_second_t> close4PointSpeed3
     {
       1_mps,
-      1.5_mps,
-      0_mps,
+      2_mps,
+      1_mps,
       0_mps
     };
 
     std::vector<units::meters_per_second_t> close4CruiseSpeed1
     {
-      2_mps,
+      2.5_mps,
       1.5_mps
     };
 
     std::vector<units::meters_per_second_t> close4CruiseSpeed2
     {
-      1_mps,
-      2_mps,
+      1.5_mps,
       1.5_mps
     };
 
     std::vector<units::meters_per_second_t> close4CruiseSpeed3
+    {
+      1.5_mps,
+      2_mps,
+      1_mps,
+      0.5_mps
+    };
+
+
+
+
+
+    // RED FRONT 4 NOTE - TODO: RED SIDE NEEDS TO BE TESTED
+    std::vector<frc::Pose2d> redClose4Waypoint1
+    {
+      frc::Pose2d(14.7_m, 5.55_m, 180_deg),
+      frc::Pose2d(13.35_m, 5.55_m, 180_deg)
```

Comment:
if these red angles don't work rotation wise could try and change it to a 0_deg?

Reply:
I don't think we need to change it to 0 degrees. I think red and blue side need to be a 180 rotation. I don't think our numbers need to reflect the 'other side of the field'.

If things work the way I think they do, if blue is (.45, 2.5, 130) then red should be (.45, 2.5, 230).

If all points are absolute based on a given starting point, then 'red' side or 'blue' side is meaningless to the robot. 
We're basically saying - given this starting point, drive here with this rotation.

You can try and get it to work with the 'inverse' of 16.55. ex blue .45 and red 16.1. But I couldn't get it to work that way and have the heading still work and have things go the correct direction. This could be a me thing.